tasks/comment/InfracostComment/index.ts

```diff
@@ -0,0 +1,134 @@
+import taskLib = require('azure-pipelines-task-lib/task');
+import compost from '@infracost/compost';
+import { PostBehavior, TargetType } from '@infracost/compost';
+import { promises as fsPromises } from 'fs';
+
+/**
+ * generateOutput() creates a formatted comment body and saves it to a temporary
+ * file. Returns the file path.
+ *
+ * @param path - path input
+ * @param format - comment format
+ * @param behavior - post behavior, used by `infracost output` command
+ * @return the saved file path
+ */
+function generateOutput(path: string, format: string, behavior: string): string {
+  const outputFile = '/tmp/infracost-comment.md';
+
+  let pathList: string[] = [path];
+  try {
+    pathList = JSON.parse(path);
+  } catch { }
```

Comment:
might be useful to add a log warning line here instead of the empty catch

Reply:
The error is expected, as path can be a string. Was not sure if we need any debug/warning message. Decided to go with this: https://github.com/infracost/infracost-azure-devops/pull/11/files#diff-54b9b23a548b3212b9c04189bc051beb211ffc6917ea5e1c82d4523e90ab331eR22